database/drivers/mysql/parse.go

```diff
@@ -15,9 +15,22 @@ import (
 	"gnorm.org/gnorm/database/drivers/mysql/gnorm/tables"
 )
 
+// MySQL implements drivers.Driver interface for MySQL database
+type MySQL struct {
+}
```

Comment:
as with postgres, let's make the type a single line and period at the end of the comment, please.